src/Beberlei/Metrics/Collector/Graphite.php

```diff
@@ -84,7 +95,14 @@ public function flush()
         }
 
         try {
-            $fp = fsockopen($this->protocol.'://'.$this->host, $this->port);
+            $fp = fsockopen(
```

Comment:
Obviously I don't get to understand and I'd love to understand what motivates the one-line thing.

Thank you for your patience @lyrixx.